kolibri/core/package.json

```diff
@@ -13,7 +13,6 @@
     "css-element-queries": "1.0.1",
     "d3-color": "^1.2.3",
     "date-fns": "^1.28.2",
-    "filesize": "^3.3.0 ",
```

Comment:
I see filesize removed here, but upgraded in the yarn.lock - is it still being used somewhere else?

Reply:
It's used in the webpack bundle analyzer, according to `yarn why filesize`